client/src/utils/imageData.js

```diff
@@ -0,0 +1,12 @@
+const BASE_URL = process.env.REACT_APP_BASE_URL;
+
+let imageData = [
+  BASE_URL + "/data/image1.jpeg",
+  BASE_URL + "/data/image2.jpeg",
+  BASE_URL + "/data/image3.jpeg",
+  BASE_URL + "/data/image4.jpeg",
+  BASE_URL + "/data/image5.jpeg",
+  BASE_URL + "/data/image6.jpeg",
```

Comment:
All these images you have added are not HD. In my laptop it looks blury. Can we get 4k images ? need better quality @Gangotri-sg3